hackclub.com.yaml

```diff
@@ -1,4327 +1,4485 @@
 nephthys:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: a.selfhosted.hackclub.com.
 neohack:
-   ttl: 600
-   type: CNAME
-   value: a.selfhosted.hackclub.com.
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 nestm:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 network.scrapbook:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 neighborhood:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 express.neighborhood:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: a.selfhosted.hackclub.com.
 newcanaan:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 newcanaanlib:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
-# explanation - news is the user-facing domain, newspaper is for internal stuff (very confusing i know :pf:)
 news:
-  - ttl: 600
-    type: ALIAS
-    value: 886e7f7579941754.vercel-dns-016.com.
-  - ttl: 3600
-    type: TXT
-    values:
-      - google-site-verification=Mc08Mplkv-e0eNBR0VW3ogoctBoozEETwZvrb-9prb0
+- ttl: 300
+  type: CNAME
+  value: 886e7f7579941754.vercel-dns-016.com.
 newspaper:
-  - ttl: 600
-    type: CNAME
-    value: a.selfhosted.hackclub.com.
+- octodns:
+    cloudflare:
+      proxied: true
+  ttl: 300
+  type: CNAME
+  value: a.selfhosted.hackclub.com.
 nigeria:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: cname.vercel-dns.com.
 nioflb:
-  - ttl: 600
-    type: A
-    value: 190.106.131.222
+- ttl: 300
+  type: A
+  value: 190.106.131.222
 nlcs:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 nmit:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
-notebook:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
+notebook:
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 ntp:
-  - ttl: 600
-    type: CNAME
-    value: ntp-hackclub.github.io.
+- ttl: 300
+  type: CNAME
+  value: ntp-hackclub.github.io.
 numberwang:
-  ttl: 600
+  ttl: 300
   type: CNAME
   value: impactdns.techby.org.
 nust:
-  - ttl: 600
-    type: CNAME
-    value: nusthackclub.netlify.app.
+- ttl: 300
+  type: CNAME
+  value: nusthackclub.netlify.app.
 oasis:
-  - ttl: 600
-    type: CNAME
-    value: cname.vercel-dns.com.
+- ttl: 300
+  type: CNAME
+  value: cname.vercel-dns.com.
 ocsa:
-  - ttl: 3600
-    type: TXT
-    values: 
-    - google-site-verification=tk_snQuv19iUwoOlsbKZv2Wwg5stfcusEm1Q8NxDiVA
-    - v=spf1 include:_spf.google.com ~all
- 
-  - ttl: 3600
-    type: MX
-    values:
-      - exchange: aspmx.l.google.com.
-        preference: 1
-      - exchange: alt1.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt2.aspmx.l.google.com.
-        preference: 5
-      - exchange: alt3.aspmx.l.google.com.
-        preference: 10
-      - exchange: alt4.aspmx.l.google.com.
-        preference: 10
```

Comment:
This MX record should not have been deleted! This is a breaking change, was this communicated to the team responsible?